### ISSUE TITLE
Support fingerprint unlocking for full-disk-encrypted configurations

### ISSUE THREAD

**Issue by anatol** [anatol/booster#15]
Some laptops (e.g. Lenovo Thinkpad) have fingerprint sensors. It would be great to integrate it with `booster`.

This issue depends on https://github.com/anatol/clevis.go/issues/2

**Comment by c3Ls1US** [comment]
I agree this would be a great and sensible feature to have as modern laptops come with TPM2 chips and fingerprint sensors. However, there doesn't seem to be a pure Go library for this. Therefore, whether in Clevis or in Booster, we'd have to wrap libfprint which circles back to requiring Go plugins or linking to it.

See https://github.com/anatol/booster/issues/101
